pandas/tests/series/indexing/test_getitem.py

```diff
@@ -147,6 +147,17 @@ def test_getitem_intlist_multiindex_numeric_level(self, dtype, box):
         with pytest.raises(KeyError, match="5"):
             ser[key]
 
+    @pytest.mark.parametrize("dtype", [np.uint8, np.uint16, np.uint32, np.uint64])
```

Comment:
can you create a fixture that is all numpy + all ea int dtypes (maybe call it INT_INDEXING or something) and use it here

Reply:
updated+greenish